app/bundles/EmailBundle/Model/EmailModel.php

```diff
@@ -1131,19 +1131,21 @@ public function &getEmailSettings(Email $email, $includeVariants = true)
      */
     public function sendEmail(Email $email, $leads, $options = [])
     {
-        $listId              = (isset($options['listId'])) ? $options['listId'] : null;
-        $ignoreDNC           = (isset($options['ignoreDNC'])) ? $options['ignoreDNC'] : false;
-        $tokens              = (isset($options['tokens'])) ? $options['tokens'] : [];
-        $assetAttachments    = (isset($options['assetAttachments'])) ? $options['assetAttachments'] : [];
-        $customHeaders       = (isset($options['customHeaders'])) ? $options['customHeaders'] : [];
-        $emailType           = (isset($options['email_type'])) ? $options['email_type'] : '';
-        $isMarketing         = (in_array($emailType, ['marketing']) || !empty($listId));
-        $emailAttempts       = (isset($options['email_attempts'])) ? $options['email_attempts'] : 3;
-        $emailPriority       = (isset($options['email_priority'])) ? $options['email_priority'] : MessageQueue::PRIORITY_NORMAL;
-        $messageQueue        = (isset($options['resend_message_queue'])) ? $options['resend_message_queue'] : null;
-        $returnErrorMessages = (isset($options['return_errors'])) ? $options['return_errors'] : false;
-        $channel             = (isset($options['channel'])) ? $options['channel'] : null;
-        $dncAsError          = (isset($options['dnc_as_error'])) ? $options['dnc_as_error'] : false;
+        $listId               = (isset($options['listId'])) ? $options['listId'] : null;
+        $ignoreDNC            = (isset($options['ignoreDNC'])) ? $options['ignoreDNC'] : false;
+        $tokens               = (isset($options['tokens'])) ? $options['tokens'] : [];
+        $assetAttachments     = (isset($options['assetAttachments'])) ? $options['assetAttachments'] : [];
+        $customHeaders        = (isset($options['customHeaders'])) ? $options['customHeaders'] : [];
+        $emailType            = (isset($options['email_type'])) ? $options['email_type'] : '';
+        $isMarketing          = (in_array($emailType, ['marketing']) || !empty($listId));
+        $emailAttempts        = (isset($options['email_attempts'])) ? $options['email_attempts'] : 3;
+        $emailPriority        = (isset($options['email_priority'])) ? $options['email_priority'] : MessageQueue::PRIORITY_NORMAL;
+        $messageQueue         = (isset($options['resend_message_queue'])) ? $options['resend_message_queue'] : null;
+        $returnErrorMessages  = (isset($options['return_errors'])) ? $options['return_errors'] : false;
+        $channel              = (isset($options['channel'])) ? $options['channel'] : null;
+        $dncAsError           = (isset($options['dnc_as_error'])) ? $options['dnc_as_error'] : false;
+        $immediately          = (isset($options['immediately'])) ? $options['immediately'] : false;
```

Comment:
I don't see any reason why to move all those line one space to the right. Can you revert it please? It will make code reviews easier if there won't be any unnecessary changes.